entities.py

```diff
@@ -46,16 +48,20 @@ class Actor(Entity):
     An actor is an entity that can perform actions.
     """
 
-    def __init__(self, name, pos, behavior, char, color):
+    def __init__(self, name, pos, behavior, char, color, blocks=True):
```

Comment:
dunno about the usefulness of this being a parameter for actors, they should all block passage, I'd just have hardcoded it in the super call, but I guess it's not too intrusive to keep

Reply:
What about corpses then, should we make a new entity subclass for them?

Reply:
Corpses shouldn't be a problem unless we want to instantiate actors as corpses no? For corpses we can just set the `blocks` flag to `False`

Reply:
OK I'll change that then.